kernel/include/devices/Rtc.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define CMOS_ADDRESS_REGISTER   0x70
```

Comment:
I try to keep all the platform specific defines in a single file (Platform.h), could you move those there and use the same naming scheme as the rest? Something like:
`PORT_CMOS_ADDRESS` and `PORT_CMOS_DATA`

Reply:
just bumping this comment